src/components/View/index.js

```diff
@@ -39,6 +39,7 @@ class View extends Component {
     const {
       hitSlop,
       style,
+      domRef,
```

Comment:
?

Reply:
OK. I don't think I will accept a patch that hacks around what is apparently another issue though